--- sbopkg.SlackBuild.orig	2019-12-07 18:05:45.733689137 +0100
+++ sbopkg.SlackBuild	2021-03-12 13:31:29.274446422 +0100
@@ -26,7 +26,7 @@
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
 rm -rf $PRGNAM-$VERSION
-tar xvf $CWD/$PRGNAM-$VERSION.tar.gz
+tar xvf $CWD/$PRGNAM-$VERSION.tar.?z
 cd $PRGNAM-$VERSION
 chown -R root:root .
 find -L . \
@@ -36,23 +36,23 @@
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
 mkdir -p $PKG/etc/sbopkg $PKG/etc/logrotate.d/
-cp -a etc/sbopkg/* $PKG/etc/sbopkg
-cp -a etc/logrotate.d/sbopkg $PKG/etc/logrotate.d/
+cp -a src/etc/sbopkg/* $PKG/etc/sbopkg
+cp -a src/etc/logrotate.d/sbopkg $PKG/etc/logrotate.d/
 mv $PKG/etc/sbopkg/sbopkg.conf $PKG/etc/sbopkg/sbopkg.conf.new
 mv $PKG/etc/sbopkg/blacklist $PKG/etc/sbopkg/blacklist.new
 
 mkdir -p $PKG/usr/sbin
-cp usr/sbin/* $PKG/usr/sbin
+cp src/usr/sbin/* $PKG/usr/sbin
 
 mkdir -p $PKG/usr/libexec/$PRGNAM
-cp -ar usr/libexec/$PRGNAM/* $PKG/usr/libexec/$PRGNAM
+cp -ar src/usr/libexec/$PRGNAM/* $PKG/usr/libexec/$PRGNAM
 
 mkdir -p $PKG/usr/man
-cp -a usr/man/* $PKG/usr/man
+cp -a src/usr/man/* $PKG/usr/man
 find $PKG/usr/man -type f -exec gzip -9 {} \;
 
 mkdir -p $PKG/usr/doc/$PRGNAM-$VERSION
-cp -a usr/doc/* $PKG/usr/doc/$PRGNAM-$VERSION
+cp -a src/usr/doc/* $PKG/usr/doc/$PRGNAM-$VERSION
 ln -s /usr/doc/$PRGNAM-$VERSION/README-repos $PKG/etc/$PRGNAM/repos.d/README
 ln -s /usr/doc/$PRGNAM-$VERSION/README-renames \
     $PKG/etc/$PRGNAM/renames.d/README
